util/llpcEmuLib.h

```diff
@@ -42,6 +42,20 @@ class Function;
 class Module;
 } // llvm
 
+namespace std
+{
+    template<> struct hash<llvm::StringRef>
```

Comment:
Please change to this style

```
namespace std
{

template<> ...
{
    ...
}

} // std
```

Reply:
I've fixed up the formatting.  Do you have a tools like clang-format that you use?  If so, which style or is there a style file that you use?

Reply:
Currently, we do not have tools like clang-format. We decide to add it later on for review convenience. There is an internal coding convention documenting coding style and detailed writing. We will discuss this and maybe publish it with LLPC project.